Readme.md

```diff
@@ -37,6 +37,10 @@ There are a couple differences when you import from NuGet instead of a Git submo
 - Use a special import call to the base style sheet in the package ([see here](https://github.com/Lombiq/Orchard-Base-Theme/blob/f2377271063d31d700cba593917195547c1654aa/Lombiq.BaseTheme.Samples/Assets/Styles/site.scss#L7-L13)). Unfortunately your IDE won't be able to reference the variables and mixins defined in the Lombiq.BaseTheme Sass files.
 - Include bootstrap in your theme file's dependencies as it's not included in the NuGet package ([see here](https://github.com/Lombiq/Orchard-Base-Theme/blob/f2377271063d31d700cba593917195547c1654aa/Lombiq.BaseTheme.Samples/package.json#L7-L9)). This actually lets you select the latest version if you need some new feature. Any 5.x should work.
 
+## Demo video
+
+[![Watch the video](https://img.youtube.com/vi/9DjKxEumoRE/maxresdefault.jpg)](https://youtu.be/9DjKxEumoRE)
```

Comment:
Nice practice about embedding the thumbnail. The YouTube image is not guaranteed to stay there forever though, and there are also privacy implications of loading it here. Please include it in the repo.

Also, I think the demo video section can be just below About.